src/plugins/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -7,16 +7,17 @@
  */
 
 import React, { useEffect, useState, useCallback } from 'react';
-import { withRouter, RouteComponentProps } from 'react-router-dom';
+import { withRouter, RouteComponentProps, useLocation } from 'react-router-dom';
+import styled from 'styled-components';
```

Comment:
I don't think we use this often. Afaik Kibana wide we are migrating to emotion (not 100% sure about the state of things). 

I think this should better be scss style or simple inline style

Reply:
fixed.